### PR TITLE
chore(docs): clarify how taginclude/tagexclude work with processors

### PR DESCRIPTION
**Pull request by redbaron**
# Required for all PRs

- [x] Updated associated README.md.
- [ ] Wrote appropriate unit tests.
- [x] Pull request title or commits are in [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: docs/CONFIGURATION.md [hunk 1]
```diff
@@ -658,12 +658,15 @@ An array of [glob pattern][] strings.  Only tags with a tag key matching one of
 the patterns are emitted.  In contrast to `tagpass`, which will pass an entire
 metric based on its tag, `taginclude` removes all non matching tags from the
 metric.  Any tag can be filtered including global tags and the agent `host`
-tag.
+tag. When used with processor plugin, metric is be modified before it is passed
```
Comment (powersj) [inline docs/CONFIGURATION.md:661]: I believe this is true of [processors](https://github.com/influxdata/telegraf/blob/master/models/running_processor.go#L90), [aggregators](https://github.com/influxdata/telegraf/blob/master/models/running_aggregator.go#L136), and [outputs](https://github.com/influxdata/telegraf/blob/master/models/running_output.go#L142). In that they all filter before processing.

I think it is worth being explicit that the filters are running after an input collects metrics but before other plugins are run.

Comment (Hipska) [reply docs/CONFIGURATION.md:661]: See also this RFC https://github.com/influxdata/telegraf/issues/8792 where I asked to modify after it has been processed by the plugin..

**Review by Hipska — CHANGES_REQUESTED** [review]
If you also add the same info for fieldpass/fielddrop, then it could partially resolve #8792.

**Review by powersj — APPROVED** [review]

**Review by srebhan — APPROVED** [review]
Thanks @redbaron for taking care!

**Pull request merged**